spec/system/data_migration/femtosecond_form_submission_spec.rb

```diff
@@ -99,6 +126,10 @@
       femtosecond_work = Work.last
       expect(femtosecond_work.title).to eq title
 
+      # Check that RORs were persisted as funder names
+      funders = femtosecond_work.resource.funders.map(&:funder_name).uniq
+      expect(funders).to contain_exactly("United States Department of Energy", "National Science Foundation", "Deutsche Forschungsgemeinschaft")
```

Comment:
Normally, we want tests to avoid dependencies on outside services, but this might be an exception: If the ROR API turns out to be flaky, it might be good to surface that, even if could mean sporadic test failures.